### PR TITLE
doc: updated 54H cs1 migration guide and gs.

### PR DESCRIPTION
**Pull request by FrancescoSer**
Updated cs1 migration guide.
Updated 54H getting started.

### CODE REVIEW

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst [hunk 1]
```diff
@@ -229,9 +229,9 @@ To program the Secure Domain Firmware (SDFW, also known as ``urot``) and the Sys
    .. note::
       On MacOS, ensure that the ZIP file is not unpacked automatically upon download.
 
-#. Move the :file:`.zip` bundle to a folder of your choice, then run nRF Util to program the binaries using the following command::
+#. Move the bundle's :file:`.zip` archive to a folder of your choice, unzip the archive, then run nRF Util to program the :file:`nordic_top.suit` using the following command::
```
Comment (annwoj) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:232]:
```suggestion
#. Move the bundle's :file:`.zip` archive to a folder of your choice, unzip the archive, and then run nRF Util to program the :file:`nordic_top.suit` using the following command::
```

File: doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs1.rst [hunk 2]
```diff
@@ -59,6 +59,6 @@ SDFW and SCFW firmwares
      .. note::
         On MacOS, ensure that the ZIP file is not unpacked automatically upon download.
 
-  #. Move the :file:`.zip` bundle to a folder of your choice, then run nRF Util to program the binaries using the following command::
+  #. Move the bundle's :file:`.zip` archive to a folder of your choice, unzip the archive, then run nRF Util to program the :file:`nordic_top.suit` using the following command::
```
Comment (annwoj) [inline doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7_99-cs1.rst:62]:
```suggestion
  #. Move the bundle's :file:`.zip` archive to a folder of your choice, unzip the archive, and then run nRF Util to program the :file:`nordic_top.suit` using the following command::
```

**Review by annwoj — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]
Generic file types should be written as per https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120291747/Typographical+conventions , so `the bundle's ZIP archive` and not `the bundle's :file:`.zip` archive`.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst [hunk 3]
```diff
@@ -229,9 +229,9 @@ To program the Secure Domain Firmware (SDFW, also known as ``urot``) and the Sys
    .. note::
       On MacOS, ensure that the ZIP file is not unpacked automatically upon download.
 
-#. Move the :file:`.zip` bundle to a folder of your choice, then run nRF Util to program the binaries using the following command::
+#. Move the bundle's :file:`.zip` archive to a folder of your choice, unzip the archive, and then run nRF Util to program the :file:`nordic_top.suit` using the following command::
 
-      nrfutil device x-provision-nrf54h --firmware <path-to_bundle_zip_file> --serial-number <serial_number>
+        nrfutil device x-suit-dfu --firmware nordic_top.suit --serial-number <serialnumber>
```
Comment (bjorgan) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:234]: The old description was correct (run x-provision-nrf54h with the zip file), the new one is incorrect.

**Review by bjorgan — DISMISSED** [review]

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst (hunk 3 above)
Comment (FrancescoSer) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:234]: @Tiwalun @kemegard Can you clarify this?

Comment (Tiwalun) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:234]: @bjorgan is corrct, if you want to install the secure domain FW on a fresh device, you need to use the `x-provision-nrf54h` command, with the zip file.

**Comment by FrancescoSer** [comment]
@kemegard @bjorgan @Tiwalun Please rereview

**Review by bjorgan — APPROVED** [review]
LGTM

@FrancescoSer: @Tiwalun was busy earlier this morning and is gone for the rest of the week now, but asked me to take care of reviewing this instead of him, so you probably don't have to wait for his approval

**Pull request merged**